packages/common/src/lib/image/secure-image.pipe.ts

```diff
@@ -20,6 +23,11 @@ export class SecureImagePipe implements PipeTransform {
       activityInterceptor: 'false'
     });
 
+    const regexDepot = new RegExp(this.configService?.getConfig('depot.url') + '.*?(?="|$)');
```

Comment:
.match() returns an array, so even if there's only one match (could be multiple), you have to extract the first (and only match in this case, since there's only one image)